src/climate_learn/metrics/functional.py

```diff
@@ -5,6 +5,7 @@
 # Local application
 from .metrics import *
 from .utils import MetricsMetaInfo
+from ..transforms import Denormalize
```

Comment:
Is it necessary to import this class because we are not really calling it in this file?